lib/Config.ts

```diff
@@ -47,6 +49,7 @@ class Config {
 
     // default configuration
     this.initDb = true;
+    this.loglevel = process.env.NODE_ENV === 'production' ? Level.INFO : Level.VERBOSE;
```

Comment:
Oh did you mean verbose for the default non-production logger levels? I'd prefer keeping it to default to DEBUG for testing/development. Eventually, certainly by beta, we should have the default NODE_ENV be production though.